api/db/seeds/data/team-certification/data-builder.js

```diff
@@ -46,7 +46,12 @@ async function teamCertificationDataBuilder({ databaseBuilder }) {
   await _createSession({ databaseBuilder });
   await _createV3Session({ databaseBuilder });
   await _createPublishedSession({ databaseBuilder });
-  await _createStartedSession({ databaseBuilder });
+  await _createStartedSession({
+    databaseBuilder,
+    sessionId: STARTED_SESSION_ID,
+    certificationCenterId: PRO_CERTIFICATION_CENTER_ID,
+    organizationId: PRO_ORGANIZATION_USER_ID,
+  });
```

Comment:
histoire d'être Iso avec tous les autres builder

```suggestion
 await _createStartedSession({
    databaseBuilder,
  });

Reply:
Par convention les méthodes  ```_createBidule``` ne prennent que le databaseBuilder en paramètre